app/jobs/pusher_call_job.rb

```diff
@@ -0,0 +1,7 @@
+class PusherCallJob < ApplicationJob
```

Comment:
Let's rename the class to `ActionCableBroadcastJob` since we aren't using pusher anymore.